### DISCUSSION TITLE
About adding Rijndael 256-bits block size

### DISCUSSION THREAD

**Discussion by superk589** [krzyzanowskim/CryptoSwift#945] [General]
In CryptoSwift, AES has a fixed block size of 128 bit and not support Rijndael 256-bits block size. any thoughts on supporting this?

**Comment by krzyzanowskim** [comment]
Actually,  I had a conversation about it recently. It should be fairly easy to add, not as AES, though (because it's not AES). With the subclass maybe (It may impact the performance). PR is welcome.
